README.md

```diff
@@ -103,6 +103,14 @@ Below are the Semantic UI components that have custom styles applied to them:
     <td>High</td>
     <td>High</td>
   </tr>
+  <tr>
+    <td><a href="https://react.semantic-ui.com/collections/table/" target="_blank">Table</a></td>
+    <td>Collection</td>
+    <td>table.css</td>
+    <td>Low</td>
```

Comment:
I think both _Usage_ and _Criticality_ can be set to high because it's a component that's being used quite a bit.

Reply:
I just thought that since they're actually using another component from the library Mike shared with us, the `Table` component alone was not critical to the application.